xcode/Subconscious/Shared/Models/Slashlink.swift

```diff
@@ -118,7 +118,7 @@ public struct Slashlink:
     }
 
     /// Convenience initializer that creates a link to `@user/_profile_`
-    init(petname: Petname) {
```

Comment:
Nit: I think I would personally prefer to keep this arg name unchanged. Reason: `petname:` mirrors the `Peer.petname` naming.

Reply:
I see this argument. I felt it was slightly confusing for the `Slashlink(petname:)` constructor to implicitly return a profile link despite saying nothing about profiles in the name, because of the more common `Slashlink(petname:slug:)`.

Happy to roll it back regardless.